code/game/turfs/snow.dm

```diff
@@ -47,16 +46,16 @@
 		M.visible_message(span_notice("\The [M] clears out \the [src]."), \
 		span_notice("We clear out \the [src]."), null, 5)
 		slayer = 0
-		update_icon(1, 0)
+		update_icon(TRUE, FALSE)
 
-	//PLACING/REMOVING/BUILDING
+//PLACING/REMOVING/BUILDING
 /turf/open/floor/plating/ground/snow/attackby(obj/item/I, mob/user, params)
 	. = ..()
 	//Light Stick
 	if(istype(I, /obj/item/lightstick))
 		var/obj/item/lightstick/L = I
 		if(locate(/obj/item/lightstick) in get_turf(src))
-			to_chat(user, "There's already a [L]  at this position!")
+			to_chat(user, "There's already a [L.name] at this position!")
```

Comment:
this shouldn't be needed.

Reply:
It is because you get `There's already a the blue lightstick at this position!` otherwise. Speaking of which noticed another one of these.

Reply:
The wording probably isn't correct then. "There's already a blue lightstick [captainisacondomhonkhonk] at this position" makes less sense than just saying "There's already a lightstick at this position", since the colour isn't actually relevant. This is giga minor though.